src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -188,9 +228,13 @@ def get_regular_task(queue=None):
   application_id = utils.get_application_id()
   while True:
     messages = _get_messages(pubsub_client, application_id, queue)
-
     if not messages:
       return None
+    task, retry = handle_single_message(messages)
+    if task:
+      return task
+    if not retry:
+      return None
 
     try:
```

Comment:
can we delete all the lines that follow here? These seem all handled in `handle_single_message`.